siibra/core/region.py

```
@@ -179,6 +179,8 @@ def parcellation(self):
     def species(self):
         # lazy request of the root parcellation's species
         if self._species_cached is None:
+            if self.parcellation is None:
+                return None
```

this looks like an inconsistent state to me. Shouldn't we always have a species defined, and do we accept regions not linked to a parcellation?  I would rather raise an Exception if that happens.

---

TBH I do not remember why I implemented this. I will check